src/PhpSessionPersistence.php

```diff
@@ -191,7 +192,10 @@ private function startSession(string $id, array $options = []) : void
      */
```

Comment:
Please update phpdoc description as it no longer reflects what is done here.

Reply:
Specifically reference to example is not relevant here.

Reply:
@Xerkus I've removed the reference link